code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm

```diff
@@ -0,0 +1,328 @@
+	var/regen_points_max = 100 // max number of points they can accumulate
+	var/regen_points_per_tick = 1 // gain one regen point per tick
+	var/regen_points_per_kill = 100 // gain extra regen points if you kill something
+	var/regen_points_per_hp = 2 // every X regen points = 1 health point you can regen
+	// desired: 30hp/minute unmolested, 60hp/min on food boost, assuming one tick every 2 seconds
+	//          100/kill means bonus 50hp/kill regenerated over the next 1-2 minutes
+
+	// Vision
+	idle_vision_range = 10
+	aggro_vision_range = 10
+	see_in_dark = 10
+	nightvision = 1
+	vision_type = new /datum/vision_override/nightvision/thermals/ling_augmented_eyesight
+	see_invisible = 5
+
+	// player control by ghosts
+	var/ai_playercontrol_allowingeneral = 1 // if 0, no spiders are player controllable. Default set in code, can be changed by queens.
+	var/ai_playercontrol_allowtype = 1 // if 0, this specific class of spider is not player-controllable. Default set in code for each class, cannot be changed.
+
+	var/spider_opens_doors = 1 // all spiders can open firedoors (they have no security). 1 = can open depowered doors. 2 = can open powered doors
+	faction = list("terrorspiders")
+	var/spider_awaymission = 0 // if 1, limits certain behavior in away missions
+	var/spider_role_summary = "UNDEFINED"
+	var/spider_placed = 0
+
+	// AI variables designed for use in procs
+	var/atom/cocoon_target // for queen and nurse
+	var/obj/machinery/atmospherics/unary/vent_pump/entry_vent // nearby vent they are going to try to get to, and enter
+	var/obj/machinery/atmospherics/unary/vent_pump/exit_vent // remote vent they intend to come out of
+	var/obj/machinery/atmospherics/unary/vent_pump/nest_vent // home vent, usually used by queens
+	var/fed = 0
+	var/travelling_in_vent = 0
+	var/busy = 0 // leave this alone!
+	var/spider_tier = TS_TIER_1 // 1 for red,gray,green. 2 for purple,black,white, 3 for prince, mother. 4 for queen, 5 for empress.
+	var/attackstep = 0
+	var/attackcycles = 0
+	var/mylocation = null
+	var/chasecycles = 0
+	var/last_cocoon_object = 0 // leave this, changed by procs.
+	var/killcount = 0
+	var/hasdroppedloot = 0
+
+	// Breathing, Pressure & Fire
+	// - No breathing / cannot be suffocated (spiders can hold their breath, look it up)
+	// - No pressure damage either - they have effectively exoskeletons
+	// - HOWEVER they can be burned to death!
+	// - Normal SPACE spiders should probably be immune to SPACE too, but meh, we try to leave the base spiders alone.
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxbodytemp = 1500
+	heat_damage_per_tick = 5 //amount of damage applied if animal's body temperature is higher than maxbodytemp
+
+	// Xenobio Interactions
+	sentience_type = SENTIENCE_OTHER // prevents people from using a sentience potion on a TS to tame it
```

Comment:
Removes this, this isn't how `sentience_type` defines work; simple animals are blacklisted by default.